src/daycare.c

```diff
@@ -742,44 +742,36 @@ void RejectEggFromDayCare(void)
     RemoveEggFromDayCare(&gSaveBlock1Ptr->daycare);
 }
 
+static const struct {
+  u16 noIncenseSpecies;
+  u16 item;
+  u16 incenseSpecies;
```

Comment:
what about `parentSpecies` instead of `noIncenseSpecies` and `babySpecies` instead of `incenseSpecies`?

Reply:
As @mrgriffin commented
> Also, I'm not sure about my choice of parentSpecies as a name, because that sounds like it would be what the mother or father are, but e.g., mother and father could both be Raichu and yet parentSpecies is Pikachu

Reply:
`currSpecies` and `babySpecies`?

Reply:
or `currSpecies` and `newSpecies`? :eyes:

Reply:
Either baby or new species works. The former is a bit more informative to what the system does but it doesn't make a big difference